test/fixtures/source-map/basic.js

```diff
@@ -0,0 +1,7 @@
+const a = 99;
+if (true) {
+  const b = 101;
+} else {
+  const c = 102;
+}
+//# sourceMappingURL=https://http.cat/418
```

Comment:
I like this url though I think we have been moving towards less funny more URLs which we control (so maybe something on nodejs.org?)